volumes/miovision/README.md

```diff
@@ -278,15 +299,86 @@ The tables below are produced using functions explained in the [API Puller](http
+
+	a) Fist, we have to create a couple of dummy tables (`volumes_test`, `volumes_15min_tmc_test`, `volumes_15min_test`) & sequences (`volumes_volume_uid_test_seq`, `volumes_15min_tmc_volume_15min_tmc_uid_test_seq`, `volumes_15min_volume_15min_uid_test_seq`) & function (`aggregate_15_min_tmc_get_valid_movement`).
+	
+	b) **Don't forget to grant `miovision_api_bot` access permission to all those new tables!**
+	
+	c) Modify the script `intersection_tmc.py` to only pull the new intersections into the dummy table and aggregate them into 15min tmc bins and **not** do the other things such as `report_date` etc.
+	
+	d) Once we have finish pulling and aggregating the data, run the below query. 
+	```sql
+	--table with all valid movements of new intersections
+	SELECT DISTINCT intersection_uid, classification_uid, leg, movement_uid
+		INTO miovision_api.intersection_movements_new
+		FROM miovision_api.volumes_15min_tmc_test;
+		ALTER TABLE miovision_api.intersection_movements_new ADD UNIQUE (intersection_uid, classification_uid, leg, movement_uid);
+		COMMENT ON TABLE miovision_api.intersection_movements_new IS 'Unique movements for each NEW intersection by classification';
+
+	--add the old ones into the old table (no need to remove decommissioned one cause what if we need them later on)
+	INSERT INTO miovision_api.intersection_movements_new
+	SELECT *
+	FROM miovision_api.intersection_movements
+	```
+
+3) Now that all the necessary source files are ready, it's time to test run everything! Create new function by directly copying from the old one but only change the table/function name it is reading from. Modify `intersection_tmc.py` again and run it.
+
+4) Check the data pulled for the new intersections to see if you find anything weird on the data, be it at the `volumes` table or at the `volumes_15min` table or even other tables.
+
+5) Once everything is checked and the data seems decent, rename the old tables by adding a date (date of last used) at the end of the table name and rename the new tables by naming it as what the old tables used to be which is the table name currently being used by the function in the daily Airflow process. 
+
+6) Manually remove decommissioned machines' data from tables `miovision_api.volumes_15min_tmc` and `miovision_api.volumes_15min`. Dont worry about other tables that they are linked to since we have set up the ON DELETE CASCADE functionality. If the machine is taken down on 2020-06-15, we are not even aggregating any of the data on 2020-06-15 as it may stop working at any time of the day on that day.
+
+7) Then, manually insert **ONLY NEW intersections** for those dates that we have missed up until today. Then from the next day onwards, the process will pull in both OLD and NEW intersections data via the automated Airflow process.
+
+Would like to just include the exact last datetime_bin of those decommissioned camera for future references.
```

Comment:
This line could instead read "update the below table of when intersections were decommissioned"